lib/network/default.nix

```diff
@@ -0,0 +1,60 @@
+/*
+  <!-- This anchor is here for backwards compatibity -->
+  []{#sec-network}
+
+  The `lib.network` library allows you to work with IPv4 addresses in CIDR notation.
+
+  ## IPv4 {#sec-network-ipv4}
+
+  ### Structure {#sec-network-ipv4-structure}
+
+  The `lib.network.ipv4` library provides ingestion functions the create an `IPv4Address` object.
+  This is an attribute set with these values:
+
+  - `cidr`: A CIDR.
+  - `address`: An IP address.
+  - `prefixLength`: A prefix length.
+  - `subnetMask`: A subnet mask.
+
+  - [`lib.network.ipv4.fromCidrString`](#function-library-lib.network.ipv4.fromCidrString):
```

Comment:
I am not really familiar with the documentation syntax, but it looks like L19-21 content redundant with documentation of function below... Does it really need to be duplicated?

Reply:
Does this comment even get rendered?

Reply:
I think `fromCidrString` was not meant to be part of the above list.